bin/download_travis_dependencies.sh

```diff
@@ -1,34 +1,34 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -eux -o pipefail
 
 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
 
-function maybe_download_from_apache_mirror {
+function try_download_from_apache {
     local spark_tarball="${SPARK_BUILD}.tgz"
-    local apache_mirror_cgi="https://www.apache.org/dyn/closer.lua"
+    local apache_archive_prefix="https://archive.apache.org/dist"
```

Comment:
Seems that it's easier to directly download from apache archive than from the mirror.